cmd/config/config.go

```diff
@@ -305,3 +319,33 @@ func writeAlphabetizedFlags(sb *strings.Builder) {
 		sb.WriteString(fmt.Sprintf("- `%s`: %s\n", flag, cliflags.AllFlagsHelp()[flag]))
 	}
 }
+
+func outputSetAction(newFields []string) (string, error) {
+	fields := struct {
+		Items []string `json:"items"`
+	}{
+		Items: newFields,
+	}
+	fieldsJSON, _ := json.Marshal(fields)
+	output, err := output.CmdOutput("update", viper.GetString(cliflags.OutputFlag), fieldsJSON)
+	if err != nil {
+		return "", errors.NewError(err.Error())
+	}
+
+	return output, nil
+}
+
+func outputUnsetAction(newField string) (string, error) {
+	field := struct {
+		Key string `json:"key"`
+	}{
+		Key: newField,
+	}
+	fieldJSON, _ := json.Marshal(field)
+	output, err := output.CmdOutput("delete", viper.GetString(cliflags.OutputFlag), fieldJSON)
```

Comment:
i know we are deleting it from the config but `Successfully deleted project` feels a bit jarring here, especially in the context of resources like flags/projects/environments.